### PR TITLE
Adding seacat auth

### PR DESCRIPTION
**Pull request by eliska-n**
- private/public ports
- remote control probably needs two we containers because of acme challenge..?
- extra config added to https conf

### CODE REVIEW

File: Site/ASAB Maestro/Descriptors/seacat-auth.yaml [hunk 1]
```diff
@@ -7,15 +7,37 @@ define:
 descriptor:
   image: teskalabs/seacat-auth
   environment:
-  # you must use "dictionary" way of writing the env variables, not list
-    SEACAT_AUTH_PROVISIONING: {{SEACAT_AUTH_PROVISIONING}}
-    ASAB_ZOOKEEPER_SERVERS: {{ZOOKEEPER_SERVERS}}
+    SEACAT_AUTH_PROVISIONING: 0
```
Comment (ateska) [inline Site/ASAB Maestro/Descriptors/seacat-auth.yaml:10]: This needs to be commented.

File: Site/ASAB Maestro/Descriptors/seacat-auth.yaml [hunk 2]
```diff
@@ -7,15 +7,37 @@ define:
...
+      aes_key=to_be_filled_in_by_remote_control
+
+      [seacatauth:credentials:mongodb:default]
+      mongodb_uri={{MONGODB_URI}}
+      mongodb_database=auth
+      tenants=yes
+      register=no
+
+      [seacatauth:communication:email:smtp]
+      host=<mocked>
```
Comment (ateska) [inline Site/ASAB Maestro/Descriptors/seacat-auth.yaml:34]: A comment needs to be here ...

**Review by ateska — CHANGES_REQUESTED** [review]

**Comment by ateska** [comment]
Nice ;-)

**Comment by eliska-n** [comment]
I recreated mongosh script, so there is no need of any bash script and it inserts seacat auth collections.
I tried to cut the original export from mongo as much as possible, so it human readable and editable.
Most of the collections use string as "_id". However, in `c` collection the id is ObjectId, so I made a specific transformation from string to ObjectId in the script for this collection, because insertOne/insertMany methods cannot work with `{"$_oid": "<actual_id>"}` syntax as other parts of mongo do.

Are there any other collections that require this transformation? Who knows!

**Comment by eliska-n** [comment]
I don't know how to handle logging properly in the script, I ended up with prints and quiting when needed...

File: Site/ASAB Maestro/Files/mongo/script/to_upload/c.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[{
+	"_id": "58acb7acccce58ffa8b953b1",
+	"email": "",
+	"username": "admin",
+	"__password": "$2b$12$gV5pNXpjT70qxAqJ.xfEGObiaIa3znrpML4R55EOsM1RvTyGxNbi.",
```
Comment (ateska) [inline Site/ASAB Maestro/Files/mongo/script/to_upload/c.json:5]: Is there a plaintext version of this anywhere?

**Pull request merged**